### PR TITLE
Store the file size after making zip file

### PR DESCRIPTION
**Pull request by morty**

### CODE REVIEW

**Comment by davidread** [comment]
Whilst we already use a helper like this on the dataset page to make a database query to get the archival and QA info, it's not really the best way. If we merge this then the dataset page not only gets the dataset from SOLR and makes a couple of database queries, it now also needs to access the disk.

Ideally we'd add the info about the zip to the dataset in the dataset schema, so that it is included when you do package_show. The advantage is that this gets cached in SOLR. The other bonus is that you can also search by this info too. Happy to discuss this today.

**Comment by morty** [comment]
I don't think that it accesses the disk. Happy to discuss moving this forward.

File: ckanext/packagezip/helpers.py [hunk 1]
```diff
@@ -8,5 +8,10 @@ def has_packagezip(pkg):
 def packagezip_url(pkg):
     return toolkit.url_for('zipfile', name=pkg.name)
 
+def packagezip_size(pkg):
```
Comment (rossjones) [inline ckanext/packagezip/helpers.py]: Are you confident no return value is safe?  No context for me to judge. Presumably there's a None check elsewhere?

Comment (morty) [inline ckanext/packagezip/helpers.py]: This is where I'm using it. https://github.com/datagovuk/ckanext-dgu/commit/a7370c0565a652f4b724af44f8ab8fc07a864676

Would it be preferable to return 0?

**Pull request merged**